app/services/csv/create_files.rb

```diff
@@ -84,11 +77,11 @@ def initialize(append_notes_content = Csv::AppendNotesContent.new)
     @append_notes_content = append_notes_content
   end
 
-  def call(data)
-    session_title = data.notes.first["session_title"] || ""
-    filename = "notes_from_#{session_title.parameterize('_')}_#{data.session_id}__"
+  def call(notes, session_title, session_id)
+    session_title = session_title || ""
```

Comment:
I'm not sure if it's possible that the session title is nil, but we were accounting for that option before so I left it.

Reply:
If we don't have a db not null constraint unfortunately we have to